### DISCUSSION TITLE
You are trying to load env variables which are already present in your project root .env

### DISCUSSION THREAD

**Discussion by manojkumarg16** [prisma/prisma#4133] [Q&A] [answered]
Error: 
      You are trying to load env variables which are already present in your project root .env
        Root: /Users/name/projectfolder/.env
        Prisma: ./prisma/.env
        Env Conflicts:
                DATABASE_URL

      You can fix this by removing the .env file from "./prisma/.env" and move its contents to your .env file at the root "/Users/name/projectfolder/.env"

I'm using @prisma/client 2.10.1, which return this error

**Comment by ryands17** [comment, marked as answer]
Hey @manojkumarg16 👋 
Thanks for reporting this. We already have an issue for this https://github.com/prisma/prisma/issues/4098 and we are working on it :)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
